### DISCUSSION TITLE
Display of user name/id/email in Kafka UI

### DISCUSSION THREAD

**Discussion by chethan8592** [provectus/kafka-ui#4480] [General]
Wanted to know can we display user name/id/email in kafka UI for user who has logged-in to application using Azure OAuth2?

![image](https://github.com/user-attachments/assets/b5d94762-1d08-4704-bcc2-9016cfee6d54)

**Comment by Haarolean** [comment]
hi, this repo is not maintained (#4255), happy to help you here: https://github.com/kafbat/kafka-ui
